spartan/aztec-network/files/config/setup-p2p-addresses.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+
+POD_NAME=$(echo $HOSTNAME)
+
+if [ "${NETWORK_PUBLIC}" = "true" ]; then
+    # First try treating HOSTNAME as a pod name
+    NODE_NAME=$(kubectl get pod $POD_NAME -n ${NAMESPACE} -o jsonpath='{.spec.nodeName}' 2>/dev/null)
+
+    # If that fails, HOSTNAME might be the node name itself
+    if [ $? -ne 0 ]; then
+        echo "Could not find pod $POD_NAME, assuming $POD_NAME is the node name"
+        NODE_NAME=$POD_NAME
+    fi
+
+    EXTERNAL_IP=$(kubectl get node $NODE_NAME -o jsonpath='{.status.addresses[?(@.type=="ExternalIP")].address}')
```

Comment:
Warning. Our AWS nodes do not have external IP addresses. So our prod deployments will likely move to GCP.